static/js/twitter.js

```diff
@@ -8,5 +8,40 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('load', 'twitter_filter()')
   fjs.parentNode.insertBefore(js, fjs)
 })(document, 'script', 'twitter-wjs')
+
+function twitter_filter() {
+  const node = obj => obj.parentNode.parentNode.parentNode.parentNode.parentNode
+  var twitter_prefix = 20
+  var twitter_handles = []
+  var orgs = document.getElementsByClassName('org-info')
+  for (var i = 0; i < orgs.length; i++) {
+    var org_info = orgs[i]
+    for (var j = 0; j < org_info.children.length; j++) {
+      if (org_info.children[j].href) {
```

Comment:
I get it just now. So, you are seeking an `a` element in the `org_info`?

There is a better way to do that.

- Now, you have an `org_info` variable.
- Assign that `org_info` with `orgs[i].children` instead.
- You have an `HTMLCollection` from the `org_info` which is contains `HTMLCollection [h3, p, a, ...]`.
- We need to convert that things to Array first. Assign `[].slice.call(org_info)` to a new variable i.e. `children`.
- So, that `children` variable will contains `Array [h3, p, a, ...]`.
- Easily eliminate the elements using `filter` method like `children.filter(x => x.nodeName === 'A' && x.host === 'twitter.com')`. You may assign this to a new variable i.e. `twitter_account`.
- Now, there will be only one node which is contains the Twitter link.
- The username is present as a `pathname` attribute. So, let use another variable called `twitter_username` then assign with `twitter_account[0].pathname.substr(1)`. We use `substr(1)` to eliminate `/`.

Feel free to ask anyway.

Reply:
Can I ask why its advisable to convert the HTMLCollection to a standard array? Is it just good practice?

Reply:
So we can run array function like filter ;)

You can use `Array.from()` instead, more readable.

Reply:
Gonna have to add another conditional statement to check that the filter actually has matches - otherwise an error is flagged when running `twitter_url[0].pathname.substr(1)`

Reply:
Cool! My suggestion above is assuming there is exactly one `A` node with Twitter hostname. You should improve it.